flycheck.el

```diff
@@ -3051,7 +3068,17 @@ Intended for use with `next-error-function'."
   (let ((pos (flycheck-next-error-pos n reset)))
     (if pos
         (goto-char pos)
-      (user-error "No more Flycheck errors"))))
+      (let ((result (cl-remove-if
```

Comment:
Tip: adding w=1 to a github diff URL ignores spaces (e.g. https://github.com/flycheck/flycheck/pull/971/files/142c1bd73170beb48403d12f55614076b66f8f53?w=1)